### PR TITLE
changelog++

### PR DESCRIPTION
**Pull request by mladlow**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -336,6 +371,31 @@ sdk/ldaputil: use EscapeLDAPValue implementation from cap/ldap [[GH-22249](https
...
+CHANGES:
+
+* core: Bump Go version to 1.20.11.
+
+IMPROVEMENTS:
+
+* core (enterprise): Speed up unseal when using namespaces
+* secrets/pki: do not check TLS validity on ACME requests redirected to https [[GH-22521](https://github.com/hashicorp/vault/pull/22521)]
+* ui: Sort list view of entities and aliases alphabetically using the item name [[GH-24103](https://github.com/hashicorp/vault/pull/24103)]
+* ui: Update flat, shell-quote and swagger-ui-dist packages. Remove swagger-ui styling overrides. [[GH-23700](https://github.com/hashicorp/vault/pull/23700)]
```
Comment (anwittin) [inline CHANGELOG.md:386]: This shows as backported to 1.15 but isn't in the improvements section.

Comment (ldilalla-HC) [reply CHANGELOG.md:386]: its in the CL for 15.2

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,41 @@
...
+* core/quotas: Close rate-limit blocked client purge goroutines when sealing [[GH-24108](https://github.com/hashicorp/vault/pull/24108)]
+* core: Fix an error that resulted in the wrong seal type being returned by sys/seal-status while
+Vault is in seal migration mode. [[GH-24165](https://github.com/hashicorp/vault/pull/24165)]
+* replication (enterprise): disallow configuring paths filter for a mount path that does not exist
+* secrets-sync (enterprise): Fix panic when setting usage_gauge_period to none
+* secrets/pki: Do not set nextUpdate field in OCSP responses when ocsp_expiry is 0 [[GH-24192](https://github.com/hashicorp/vault/pull/24192)]
+* secrets/transit: Fix a panic when attempting to export a public RSA key [[GH-24054](https://github.com/hashicorp/vault/pull/24054)]
+* ui: Fix JSON editor in KV V2 unable to handle pasted values [[GH-24224](https://github.com/hashicorp/vault/pull/24224)]
+* ui: Fix error when tuning token auth configuration within namespace [[GH-24147](https://github.com/hashicorp/vault/pull/24147)]
+* ui: show error from API when seal fails [[GH-23921](https://github.com/hashicorp/vault/pull/23921)]
```
Comment (anwittin) [inline CHANGELOG.md:38]: line 38 is also old.

File: CHANGELOG.md (hunk 1 above)
Comment (mladlow) [reply CHANGELOG.md:386]: It was backported to 1.15 on Oct 26, and to 1.14 on Nov 13, so the changelog is correct based on those timelines. It appears in the 1.15.2 changelog.

File: CHANGELOG.md (hunk 2 above)
Comment (mladlow) [reply CHANGELOG.md:38]: Also double checked on this one - we agreed it was correct based on merge date, and expected to only appear in 1.15.

**Pull request merged**